src/main/config/config_streamer.c

```diff
@@ -53,6 +53,8 @@ extern uint8_t __config_end;
 #  define FLASH_PAGE_SIZE                 ((uint32_t)0x8000)
 # elif defined(UNIT_TEST)
 #  define FLASH_PAGE_SIZE                 (0x400)
+# elif defined(SITL)
```

Comment:
@cs8425 Sorry, I realised that my comment was not clear. I am not at all opposed to `SITL` as the name of the target. But target specific defines should be strictly restricted to `src/main/target`. Any conditional code specific to the build for a non-physical target should use a secondary define (like 'SIMULATOR_BUILD`,  albeit I am not happy with that name either), and `src/main/target/SITL/target.h` should contain a define for this.

(Historical point in case: A substantial amount of effort was put into removing all target specific conditionals from outside `src/main/target`, so we should avoid the introduction of new ones.)